canary/apps/react/cra-ts/config-overrides.js

```diff
@@ -0,0 +1,3 @@
+const { removeModuleScopePlugin } = require('customize-cra');
+
+module.exports = removeModuleScopePlugin();
```

Comment:
What is this needed for?

Reply:
Create React App won't handle imports from files outside of src. This fixes that with this `cutomize-cra` library.